content/fr/service-discovery.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Découverte de services (Service Discovery)
+status: Completed
+category: Concept
+tags: ["réseau", "", ""]
+---
+
+La découverte de services est le processus permettant de découvrir les instances individuelles qui composent un service.
+Un outil de découverte de service garde une trace des différents nœuds ou points d'extrémité qui constituent un service.
```

Comment:
Il me semble qu'on traduit généralement "endpoint" par "point de terminaison"

Reply:
fix in https://github.com/cncf/glossary/pull/2998/commits/6edac4e05b2075aace1c8a1166e3e80aacf06996